exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
```

Comment:
It doesn't seem that you ever use pandas in this document. Are the inputs pandas data frames? It might be nice to include that in the input comment.